values.yaml

```diff
@@ -7,10 +7,8 @@ nodes:
     address: 'http://substra-backend.node-1.com'
     user: 'node-1'
     password: 'p@$swr0d44'
-    shared_path: '/tmp/org-1/servermedias'
   - name: 'node-2'
     msp_id: 'MyOrg2MSP'
     address: 'http://substra-backend.node-2.com'
     user: 'node-2'
     password: 'p@$swr0d45'
-    shared_path: '/tmp/org-2/servermedias'
```

Comment:
Why remove the shared path? Is the feature of using shared path when adding data samples deprecated?

Reply:
I removed sharedPath as server medias is not mount as a hostPath pv by default. With k8s-backend, pv use now the default storage class and not hostPath.
Adding data samples using a shared path between the host and pods is not deprecated :) I just miss to add an extra value to configure it (see https://github.com/SubstraFoundation/substra-backend/pull/304) !

Reply:
The trouble is that removing this shared_path value prevents `test_add_data_sample_located_in_shared_path` from running at all (it won't be executed). Is there a way we could still run this test with the new setup?